README.md

```diff
@@ -1,5 +1,28 @@
 # EB CLI Installer
 
+## 0. Easy Install with [pipx](https://pypa.github.io/pipx/)
+
+pipx handles the isolated virtual environment creation for the `awsebcli` package.
+
+```shell
+python -m pip install pipx
+
```

Comment:
```suggestion
pipx ensurepath
```

can be added for completeness if we want to match the installation instructions from the `pipx` docs: https://pypa.github.io/pipx/installation/